elastic.go

```diff
@@ -18,16 +19,18 @@ type Handler struct {
 	typ    string
 	// Refresh sets the refresh flag to true on all write operation to ensure
 	// writes are reflected into search results immediately after the operation.
-	// Setting this parameter to true has performance impacts.
-	Refresh bool
+	// Setting this parameter to "true" has performance impacts.
+	Refresh string
```

Comment:
I'm just exposing a parameter of the elastic driver. They changed their API from bool to string for this so it can get 3 value (true, wait_for, false).